### ISSUE TITLE
Program shows fantom "Generic Hard Drive"

### ISSUE THREAD

**Issue by inliquid** [openhardwaremonitor/openhardwaremonitor#1325]
Win10 Pro
v0.9.5
There are 2 drives in the system :
1. System NVMe SSD
2. Additional HDD

Program shows 3 drives including some mysterious "Generic Hard Drive" with no attributes at all:
![изображение](https://user-images.githubusercontent.com/6797881/91638810-bed68300-ea1a-11ea-9505-701689b3b72e.png)

Here is for example how it looks like in HWMonitor:
![изображение](https://user-images.githubusercontent.com/6797881/91638868-1aa10c00-ea1b-11ea-9fac-f328e8657c93.png)

**Comment by ChelOis** [comment]
I have a somewhat similar problem. I reported it as Issue #1345.

**Comment by kevinnio** [comment]
I'm seeing the same "Generic Hard Disk" on my system. Not a major issue, but weird nonetheless.

**Comment by donpin1** [comment]
Agree, please add NVMe drives and their temperatures. It would be great if you also had a tab that monitored SMART attributes. Nevertheless, this is easily the best program for me that I have come across. Well done.
![Capture](https://user-images.githubusercontent.com/79115442/108458136-dfe8a400-72d8-11eb-97bf-d9bcc5b88f54.JPG)
